microprofile/lra/testing/src/main/java/io/helidon/microprofile/testing/lra/package-info.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Test LRA coordinator.
+ * Allows simplified testing of LRA enabled JAX-RS resources.
+ *
+ * <pre>{@code
+ * @HelidonTest
+ * @AddBean(TestLraCoordinator.class)
+ * @Path("/test")
+ * public class LraTest {
+ *
+ *     private final WebTarget target;
+ *     private final Set<String> completedLras;
+ *     private final Set<String> cancelledLras;
+ *
+ *     @Inject
+ *     private TestLraCoordinator coordinator;
+ *
+ *     @Inject
+ *     public LraTest(WebTarget target) {
```

Comment:
Please do not combine constructor and field injection, even in tests. Constructor injection is prefered.